frame/nomination-pools/src/lib.rs

```diff
@@ -1314,6 +1576,12 @@ pub mod pallet {
 	#[pallet::storage]
 	pub type MaxPoolMembersPerPool<T: Config> = StorageValue<_, u32, OptionQuery>;
 
+	/// The maximum commission that can be charged by a pool. Used on commission payouts to bound
+	/// pool commissions that are > `GlobalMaxCommission`, necessary if a future
+	/// `GlobalMaxCommission` is lower than some current pool commissions.
+	#[pallet::storage]
+	pub type GlobalMaxCommission<T: Config> = StorageValue<_, Perbill, OptionQuery>;
```

Comment:
If this is zero, we expect the previous blocks to execute exactly the same. We don't have the tools for this now, but try-runtime could help. 

We could load the new runtime, and replay some of the old blocks.